ppr-ui/src/components/mhrRegistration/YourHome/OtherInformation.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
```

Comment:
Never a bad idea to add an `id` to your components. 
Sometimes useful for unit testing/automation and/or scroll to effects that we will include with our future app-wide validations.

Reply:
Added an `id`.